src/current/v22.2/backup.md

```diff
@@ -177,6 +177,34 @@ Improve the speed of backups to Azure Storage by increasing `cloudstorage.azure.
 
 **Default:** `1`
 
+#### Cloud storage cluster settings
+
+The following cluster settings limit the read and write rates to [cloud storage]({% link {{ page.version.version }}/use-cloud-storage.md %}). A user may choose to use these settings if their backups overwhelm the network. These settings limit throughput and as a result backups will take longer. The designated `<provider>`s include `s3`, `gs`, and `azure`.     
+
+#### `cloudstorage.<provider>.write.node_rate_limit`
+
+Limit the number of bytes per second per node across operations writing to the designated cloud storage provider if non-zero.
+
+**Default:** `0 B`
+
+#### `cloudstorage.<provider>.write.node_burst_limit`
+
+Burst limit the number of bytes per second per node across operations writing to the designated cloud storage provider if non-zero.
```

Comment:
Burst is a little different than rate; I think we must have another rate limiter somewhere where we document the burst limit from which we could copy?

Reply:
changed burst rate to mention that bytes are handled concurrently